### DISCUSSION TITLE
Week picker: Adding a month name

### DISCUSSION THREAD

**Discussion by ux-git** [sircharlo/meeting-media-manager#2198] [Ideas] [answered]
### Is your idea related to a problem?

In our area (I mean Europe) it's common to use the "15 Oct 2023" date format to easily understand what month is this without additional efforts of thinking/converting numbers to month name. Thus, seeing the long list of numbers in week list is quite confusing.

### Describe your idea

Is it possible to add a month name (short forms to not change the field length) to this UI element and others that uses the date? Additionally, that's how many publishers got used to see the meeting dates from JW Library. 

![5git](https://github.com/sircharlo/meeting-media-manager/assets/80993061/1f78b7c7-be3c-4864-9569-e22a4ad6be30)

### Describe alternatives you've considered

Tried to convert the month's number into actual month name and misclicked multiple times 😄

**Comment by sircharlo** [comment, marked as answer]
I think this would be feasible for the upcoming refactor.
